src/main/java/com/ldtteam/structurize/client/gui/WindowReplaceBlock.java

```diff
@@ -165,10 +174,25 @@ public void onButtonClicked(@NotNull final Button button)
         if (button.getID().equals(BUTTON_DONE))
         {
             final ItemStack to = findPaneOfTypeByID("resourceIconTo", ItemIcon.class).getItem();
-            if (!ItemStackUtils.isEmpty(to))
+            if (!ItemStackUtils.isEmpty(to) || to.getItem() instanceof AirItem)
```

Comment:
This makes no sense.
Why are we checking here for Air?
Please document this better.

Reply:
air itemstacks are considered empty itemstacks, yep we have remove button but also we air block in listing, this is just quality of life change